src/main/webapp/WEB-INF/view/admin.jsp

```diff
@@ -38,17 +40,17 @@
 
       <h1>Administration</h1>
       <p>
-        This is the administration page of the CodeU Chat App. Only the administrators of the site can view stats. 
+        This is the administration page of the CodeU Chat App. Only the administrators of the site can view stats. (Enjoy the authority!) 
       </p>
 
       <h2><strong>Site Statistics</strong></h2>
       <ul>
-        <li><strong>Total Users:</strong></li>
-        <li><strong>Total Conversations:</strong></li>
-        <li><strong>Total Messages:</strong></li>
-        <li><strong>Most Active User:</strong></li>
-        <li><strong>Newest User:</strong></li>
-        <li><strong>Wordiest User:</strong></li>
+        <li><strong>Total Users:</strong> <%= AdminServlet.getTotalUsers() %></li>
```

Comment:
Using the approach of setting attributes on the request will let you change these to calls like `request.getAttribute(...)`.

Reply:
Ok! fixed!